nservicebus/scalability-and-ha/distributor/index.md

```diff
@@ -73,6 +73,7 @@ For more information about Pub/Sub in a distributor scenario see [What the distr
 
 ## Distributor configuration
 
+NOTE: As of Version 6 NServiceBus no longer supports running an embedded distributor. Because of that configuration sections like `MasteNodeConfig` are obsoleted. Read more about [Upgrading a Distributor-based scaled out endpoint to V6](/samples/scaleout/distributor-upgrade/)
```

Comment:
@SzymonPobiega - Do we need to rework this article or make this relevant only for versions 5 and below?
Somehow inserting a note down in the depths seems incorrect, if this is fundamentally a wrong approach to take for Versions 6 and above.
Also I think we ought to cover this in the upgrade guide. 
@WojcikMike - Not sure if adding a note here is the right approach. Let's see what @SzymonPobiega says.